platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/api/ConfigurationRoleUsageIntegrationTest.groovy

```diff
@@ -713,43 +713,86 @@ class ConfigurationRoleUsageIntegrationTest extends AbstractIntegrationSpec impl
     // endregion Migrating configurations
 
     // region Detached configurations
-    def "changing usage on detached configurations does not warn"() {
+    def "changing usage #property = #change (change property to false) on detached configurations is permitted"() {
```

Comment:
Can we add comment here saying this is not desired behavior and that ideally this should fail?